apps/pwabuilder-google-play/Next-steps-unsigned.html

```diff
@@ -0,0 +1,7 @@
+<html>
+    <body>
+        <script>
+            window.location = "https://github.com/pwa-builder/CloudAPK/blob/master/Next-steps-unsigned.md";
```

Comment:
Change this to the link on the pwabuilder repo, rather than the cloudapk (PWABuilder Google Play) repo.